### ISSUE TITLE
The Gamma Utils Night Vision interferes with the normal vanilla one

### ISSUE THREAD

**Issue by mrjasonn** [Sjouwer/gamma-utils#53]
This bug overrides the game's night vision. So, if you're on a server like Hypixel that, in some cases, gives you night vision, the mod allows you to turn off or adjust the vanilla night vision percentage. However, you should be at 100% night vision, as the vanilla night vision is applied. Additionally, when you turn off night vision with the mod but still have it server-side, you can't see your effect timer until it syncs with the server again.

Potential fix: Allow both vanilla night vision and Gamma Utils night vision to be on at the same time. Make sure you can't adjust or turn off the vanilla night vision when it's active; only adjust the Gamma Utils night vision. Ensure that having both active won't cause a double effect.

Please let me know if more clarification is needed.

**Comment by Sjouwer** [comment]
The Night Vision Gamma Utils uses is the exact same one as vanilla mc uses. There is no difference between them. That's why you experience the "interferences". I can look into better distinguishing them both by creating a separate effect. Though imo it doesn't have a high priority atm, if you use the Night Vision of Gamma Utils there isn't really a reason why you'd want to use the vanilla one.

**Comment by mrjasonn** [comment]
> The Night Vision Gamma Utils uses is the exact same one as vanilla mc uses. There is no difference between them. That's why you experience the "interferences". I can look into better distinguishing them both by creating a separate effect. Though imo it doesn't have a high priority atm, if you use the Night Vision of Gamma Utils there isn't really a reason why you'd want to use the vanilla one.

@Sjouwer Yes, please do look into better distinguishing them both by creating a separate effect, that would be a good change, however, even though it is client side, it might possibly a good idea to test joining servers with a test account (if possible) to make sure that anticheats don't false flag, even though it really shouldn't.
